src/modules/discord/commands/services/register.service.ts

```diff
@@ -20,6 +20,25 @@ export class RegisterCommandService {
       };
     }
 
+    try {
+      const user = await this.db.user.findUnique({
+        where: {
+          discordId: discordUserId,
+        },
+      });
+
+      if (user) {
+        return {
+          error: `You are already registered`,
+        };
+      }
+    } catch (e) {
+      logger.error(e);
+      return {
+        error: `Could not fetch user`,
```

Comment:
Would be nice to have details about the fetched user here `Could not fetch user ${discordUserId}`